backend/src/auth/auth.guard.ts

```diff
@@ -8,6 +8,7 @@ export class AuthGuard implements CanActivate {
 	canActivate(
 		context: ExecutionContext,
 	): boolean | Promise<boolean> | Observable<boolean> {
+		// return true;
```

Comment:
hehe was die nog van mij???

Reply:
ah noooooooo
that's mine actually.. Thought it would be easy to leave it there to uncomment/comment instead of writing it each time you want to disable the guard..